### DISCUSSION TITLE
Pixel options for screen scaling

### DISCUSSION THREAD

**Discussion by SheepCommander** [ppy/osu#24913] [Ideas]
I'm on an ultrawide (3440x1440) monitor, but I want to play at 1920x1080 to keep it consistent for when I'm away from home.

This is currently not easily achieved, as if I set the fullscreen resolution to 1920x1080 my GPU drivers stretch the game to the screen instead of padding it with black bars.

At one point I was told to "Just use the UI scaling. That's the supported way anyways." (can't remember who told me that)

But using the screen scaling options I cannot get the width right as 1920p is 55.8% of 3440p. (a non-integer percentage)

---

tl;dr pls add a "Scale screen to X Y resolution" at some point, until then I will continue playing windowed and resizing myself

**Comment by Shirou27** [comment]
I completely agree with this. What I do for now is change the resolution of my phone instead of inside the game, since it's not implemented yet in osu!lazer.

**Reply by SheepCommander** [reply]
Oh also! A temp solution for those on Windows:

1. Install "Windows PowerToys" from the github or microsoft's website
2. Open settings
3. Find "FancyZones" on the left menu
4. Enable it and launch layout editor
5. Create new layout (either option works)
6. Add a new zone and resize it to 1922 x 1112
7. Save & Apply then exit

Now if you hold shift while dragging the Osu window, you should be able to snap it to fit that zone. After you snap it, Osu should tell you that it is now 1920x1080!   `[In win11, windows have a 1 pixel margin on all sides as well as a 30 pixel header on the top, which is why the zone has to be +2 wider and +32 taller than the resolution you're aiming for]`

**Comment by yesseruser** [comment]
For now, my only solution is to use borderless on Windows. I get around 800 fps with opengl in the menu.

**Comment by cmm7878** [comment]
i need this. using screen scaling leaves slight errors since 2/3 (the difference between a monitor i just got and the laptop screen ive been using) doesnt go into decimal perfectly that are so incredibly noticeable and i dont want to have to switch between the 2 barely different aspect ratios any time i want to play on the go

**Reply by SheepCommander** [reply]
Since this post I now rock a standard QHD 3840x2160 monitor, so FHD 1080p happens to be exactly 50% of a 4k screen's size. But that's dumb and I do think it's crazy to just refuse to support this lol
